src/desktop/WLSurface.hpp

```diff
@@ -98,14 +99,15 @@ class CWLSurface {
   private:
     CWLSurface() = default;
 
-    bool                   m_bInert = true;
+    bool                         m_bInert = true;
 
-    WP<CWLSurfaceResource> m_pResource;
+    WP<CWLSurfaceResource>       m_pResource;
 
-    PHLWINDOWREF           m_pWindowOwner;
-    PHLLSREF               m_pLayerOwner;
-    CPopup*                m_pPopupOwner      = nullptr;
-    CSubsurface*           m_pSubsurfaceOwner = nullptr;
+    PHLWINDOWREF                 m_pWindowOwner;
+    PHLLSREF                     m_pLayerOwner;
+    CPopup*                      m_pPopupOwner      = nullptr;
+    CSubsurface*                 m_pSubsurfaceOwner = nullptr;
+    CXxColorManagementSurfaceV4* m_pCMSurface       = nullptr;
```

Comment:
the reason popup and subsurface are C pointers is that they haven't yet been rewritten. Please use managed pointers (a WP here)